Resources/config/security.xml

```diff
@@ -41,6 +41,10 @@
 		<service id="scheb_two_factor.provider_registry" class="%scheb_two_factor.provider_registry.class%">
 			<argument type="service" id="scheb_two_factor.session_flag_manager" />
 			<argument /> <!-- Two-factor providers -->
+			<call method="setEventDispatcher">
+				<argument type="service" id="event_dispatcher" />
+				<argument type="string" id="%auth_code%" />
```

Comment:
It should be another call, since `setEventDispatcher` accepts a single argument.

Btw, why do you even use setter injection?